### ISSUE TITLE
"One does not simply contribute to drupal"

### ISSUE THREAD

**Issue by treasuretron** [spacebase/spacebasenz#153]
In GitLab by @richbodo on May 7, 2018, 21:30

To discuss prior to V1 release:

**** Theory: Creating an organization on drupal.org might help build a positive reputation with the drupal community, and encourage collaboration and contribution on this project.  Contributions funded by spacebase dollars would be attributed to spacebase, our platform would get some exposure, and it might be found helpful by others.  A strong presence among the community of developers that work on the platform you are working with has a lot of value.

**** Fact: An unknown amount of work is required by drupal.org management rubric, prior to creating an organization on drupal.org, and Rich is not qualified to do any of that work, and doesn't want to be.  He tried to create an org, and was rejected for lack of contribution.  Fair enough.

**** Options:

1) Skip the org but stick with drupal.  This is the current track, but not ideal.  In this scenario we create the open source project, but we don't attempt to interact much with the drupal community.  Contributions can go through individual devs, and they can interact.  This makes drupal a worse choice than we thought it would be.  This has been written to discuss getting of this track.

2) Throw one away - skip the org and plan a rewrite.  In this scenario we view this as a yellow flag, and investigate the future of drupal.  Found lacking, we would ground-up rewrite in the long run, with a community we can work with.  There are definitely technical arguments to rewrite and go with another platform, so this is already on the table.  Worst case, it could be that there is more crazy in the drupal community than is worth dealing with.  We would probably get to V1.0, and maintain that software for a while while we rewrite on another platform.  This is currently our second choice.

3) Keep trying until we succeed with the org and stick with drupal.  It is *probable* that one or more people on the team, or people that we know, could help get the organization started, probably under the name of a non-staff member, which I *think* is fine?  I don't know what the ramifications of that are.  No gaurantee we can get that done.  Still, this is our first choice - it might get demoted depending on what we learn.  So, I'll record what I've done so far, and we can revisit this before V1.  

**** What I've done so far:

1) Tried to create an org.  Rejected.  Filed a ticket to discuss.  Here that is:

https://www.drupal.org/project/webmasters/issues/2958643

2) Stephen clicked some "this guy is not a spammer" button somewhere.

3) Tried to add an organization again: https://www.drupal.org/node/add/organization

Entered the following:

**Org Name:** SpaceBase

**Org Type:** Other

**Logo:** Attached

**Website:** https://spacebase.co

Locations Served: New Zealand

**Offices In:** Americas, Asia-Pacific

**Services:** None

**Sectors: **Community

**Usual Project Budget:** blank

Do NOT request listing in drupal services section.

**Organization Description:**

SpaceBase is an impact organization, based in New Zealand, writing a federated collaboration platform for the space industry, in Drupal 8.  

We are space scientists and collaboration experts working with the drupal community.  We employ four drupal developers and one designer as of this writing.

We want to contribute as much of the code we write as possible, and we are designing our platform to be fairly generic - allowing a wide variety of communities to hit the ground running with a pre-built D8 platform.

Long-term, where we are trying to innovate is in data federation - allowing many instances of the platform to search across a federation, or collaborate on shared data objects.

Our vision for the space industry:

We see a world in which all people have an opportunity to participate in the space industry on an equal footing, in accordance with their aptitude and ambitions.

We plan to contribute to this vision by making education and entrepreneurial support available to those who need it most. 

In short, we want to democratize space.  We would rather pass on the Federation than Elysium to the next generation, and we believe that outcome is best achieved when as many people as possible have the education and support required to participate in that effort.

**Drupal Contributions:**

None yet.  Creating an org so we can attribute the contributions to the org.

**Comment by treasuretron** [comment]
In GitLab by @richbodo on May 7, 2018, 21:31

closed

**Comment by treasuretron** [comment]
In GitLab by @richbodo on May 7, 2018, 22:16

It appears that whatever Stephen did (or something else) worked.  Fingers crossed.  We shouldn't have to reopen this if we still have a drupal org by the time this milestone comes up.
